app/javascript/src/components/editor/Compiler.svelte

```diff
@@ -24,11 +32,9 @@
   }
 
   function copyToClipboard(value) {
-    copied = true
-
     copyValueToClipboard(value)
 
-    setTimeout(() => copied = false, 1000)
+    addAlert("Copied compiled snippet to clipboard")
```

Comment:
I don't think it should be an alert. It is much nicer inline since that's where your eyes already are

Reply:
I have reverted back to the tooltip.
Are you still not okay with the amount of `if`s needed for the button's text? Splitting the button into an inline and not inline version is a little awkward due to the shared logic on click.

Reply:
Actually I think by now it's totally ok as is, before it was mostly with the extra logic for it being a button or a span, that it felt like it could be more readable if it was split up. But as it is now I think it's definitely readable enough, nicely done.